peerstore.go

```diff
@@ -10,6 +10,9 @@ import (
 
 var _ Peerstore = (*peerstore)(nil)
 
+const maxInternedProtocols = 256
+const maxInternedProtocolSize = 96
```

Comment:
let's make these a little larger.

Reply:
I think the sizing is fine; can't really see protocol strings being longer than 96 chars nor us handling 256 unique proto IDs over a long period. (Opinion weakly held).

Reply:
Upped. Note: I consider this to be a temporary fix. We should probably specialize protocol handling and push it down into the peerstore implementations.

Reply:
I doubt this'll be an issue but 512/256 is probably fine anyways (128KiB).